### ISSUE TITLE
Added support for innodb_buffer_pool_instances

### ISSUE THREAD

**Issue by tersmitten** [Oefenweb/ansible-percona-server#12]
http://dev.mysql.com/doc/refman/5.5/en/innodb-parameters.html#sysvar_innodb_buffer_pool_instances

**Comment by mvdriel** [comment]
In #14 it is possible to create more flexible MySQL configuration. For example:

```
percona_server_my_cnf:
  - section: mysqld
    options:
      - name: innodb_buffer_pool_instances
        value: 8
```

Does this fix this issue?

**Comment by tersmitten** [comment]
Fixed in #17
